cdap-api/src/main/java/co/cask/cdap/api/flow/FlowletDefinition.java

```diff
@@ -55,8 +57,16 @@
   private Map<String, Set<Schema>> inputs;
   private Map<String, Set<Schema>> outputs;
 
-  FlowletDefinition(String flowletName, Flowlet flowlet, int instances) {
-    FlowletSpecification flowletSpec = flowlet.configure();
+  public FlowletDefinition(String flowletName, Flowlet flowlet, int instances) {
+    FlowletSpecification flowletSpec;
+    if (flowlet instanceof AbstractFlowlet) {
```

Comment:
maybe add a todo comment with a jira to remove deprecated and replace with new configure method after next release